Ldtpd/Value.cs

```diff
@@ -303,7 +307,7 @@ public double GetMinIncrement(String windowName, String objName)
         }
         public int Increase(string windowName, string objName, int iterations)
         {
-            double max = GetMaxValue(windowName, objName);
+            double max = Double.MaxValue;//GetMaxValue(windowName, objName);
```

Comment:
Why changing this ? We need to get the max value of current object, rather than the actual double max.